[Validator] Add a constraint to sequentially validate a set of constraints (#13206)
symfony/symfony-docs: validation/sequence_provider.rst

                // ...
            }
        }

How to Sequentially Apply Constraints on a Single Property
==========================================================

> In most simple cases, you may want to apply constraints sequentially on a single

Reviewer: ```suggestion
Sometimes, you may want to apply constraints sequentially on a single
```
(same reason as above)
